onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -75,15 +76,18 @@ def fuse_xformers(model: ir.Model) -> tuple[ir.Model, dict[str, int]]:
     fusion_count["partial_rotary_embedding"] = fuse_partial_rotary_embedding(model)
     fusion_count["cos_sin_cache"] = fuse_cos_sin_cache(model)
     fusion_count["sdpa"] = fuse_sdpa(model)
+    model = _pre_optimize(model)
```

Comment:
Can you add a comment to the source why this was necessary? I suspect this may be something we need to cleanup later.